frontend/validator.js

```diff
@@ -51,18 +56,36 @@ var Validator = (function() {
         mode: 'cors',
         cache: 'no-cache',
       });
+
       fetch(req)
         .then(function(response) {
           return response.text();
         })
         .then(function(response) {
           appendLog('fetched: ' + response);
-          document.querySelector(textarea_selector).innerHTML = response;
+          // store response into the textarea, FIXME this is currently exploitable, paste a HTML page
```

Comment:
This will require some time to fix. 
It is currently only exploitable when the user pasts an untrusted URI. See #17